setup/README.md

```diff
@@ -96,19 +97,32 @@ From the **Plugin Listing** tab, click on the `Get Plugins...` button near the u
 
 ![get_plugins](resources/get_plugins.png)
 
-In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest ngld **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step).
+In the **Get Plugins** window, select the `[FFXIV+others]Overlay Plugin` option and click on `Download and Enable`. This will add the latest **OverlayPlugin** to ACT (the OverlayPlugin auto-updater may also run during this step). Note that the number displayed next to the plugin may differ from the one shown here.
 
 ![get_plugins_window](resources/get_plugins_window.png)
 
 The OverlayPlugin should now be setup. Click on the `X` to close the **Get Plugins** window.
 
 **At this point it is recommended to restart ACT before continuing on.**
 
+### Using OverlayPlugin to End Encounters
+
+It is recommended to use OverlayPlugin's in/out-of-combat detection to split encounters, rather than ACT's less accurate behavior.
+
+To do this, first set ACT's encounter split timeout (Options > Main Table/Encounters > General) to a large value, such as 60 seconds (or even higher, for duties with long cutscenes):
+
+![act timeout settings](https://user-images.githubusercontent.com/14287379/199086818-7509e165-70a8-49bc-b43c-2ccfa3a9235b.png)
```

Comment:
Can you check these images into the repo?

Reply:
Done